docs/registry/_index.md

```diff
@@ -27,18 +27,33 @@ aliases:
 menuindent: true
 ---
 
+The [Viam registry](https://app.viam.com/registry) is the central place where you can browse [ML models](/ml) to deploy with machine applications like [computer vision](/ml/vision/), or {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`.
+
+### ML models
+
+Viam provides the ability to train, upload, and deploy ML models within the platform.
+See [Machine Learning](/ml/) for more information.
+
+You can use these ML models to make detections or classifications on-machine with an [`mlmodel` vision service](/ml/vision/mlmodel/).
+
+The Viam registry hosts pretrained ML models that users have made public, so that you can save time spent training complicated models and quickly deploy functional classifiers or detectors for your use case onto your robot.
```

Comment:
```suggestion
The Viam registry hosts pretrained ML models that users have made public, so that you can save time spent training complicated models and quickly deploy classifiers or detectors for your use case onto your robot.
```